switches/switch/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_executable(asebaswitch
 	switch.cpp
 )
 
-target_link_libraries(asebaswitch ${ASEBA_CORE_LIBRARIES})
+target_link_libraries(asebaswitch ${ASEBA_ZEROCONF_LIBRARIES} ${ASEBA_CORE_LIBRARIES})
```

Comment:
The variable `ASEBA_ZEROCONF_LIBRARIES` is not defined if `DNSSD_FOUND` is false, so I think this will produce an error. In Studio, this is solved by having an `ADDITIONAL_LIBRARIES` variable that accumulates all optional library dependencies. As zeroconf will be used in different places, this is maybe too verbose. A way to do it cleanly is to define a variable called `ASEBA_OPTIONAL_ZEROCONF_LIBRARIES`, whose value is either `ASEBA_ZEROCONF_LIBRARIES` if `DNSSD_FOUND` is true, or empty otherwise.